Sources/Testing/Testing.docc/OrganizingTests.md

```diff
@@ -17,29 +17,29 @@ Organize tests into test suites.
 When working with a large selection of test functions, it can be helpful to
 organize them into test suites.
 
-A test function can be added to a test suite in one of several ways:
+A test function can be added to a test suite in one of two ways:
 
-@Comment { 0. By placing it in the same file as other test functions; }
-1. By placing it in a Swift type; or
-2. By placing it in a Swift type and annotating that type with the `@Suite`
+@Comment{ * By placing it in the same file as other test functions. }
```

Comment:
Why does this list get periods, but the other lists lose punctuation? :)

Reply:
Different types of capitalization and punctuation rules for lists and list types. When you have a list of items that complete a sentence, you initial-cap the first word and don't terminate with a period. When you have a list of items that consists of complete sentences, you initial-cap the first word and terminate with a period. Lists of random items don't terminate with a period, and bullet list items that consist of two or more sentences follow standard punctuation rules (i.e., terminate sentences with a period).

Reply:
🤔 Alrighty.

Reply:
Sorry, it's very convoluted.